### DISCUSSION TITLE
Grouping syntax

### DISCUSSION THREAD

**Discussion by vicary** [nativewind/nativewind#395] [Ideas]
We want to port a set of custom made TailwindUI components, using Twind's grouping syntax, to Nativewind.

Is it possible to support [grouping syntax](https://twind.dev/handbook/grouping-syntax.html#frontmatter-title) in twind?

**Comment by marklawlor** [comment]
No, because we use the Tailwind CLI to compile the styles. Twind is a complete different compiler

**Reply by vicary** [reply]
Nice and clear, thanks for answering.

**Comment by danstepanov** [comment]
Closing this discussion, if this is still relevant, feel free to open a new one.
